src/css/components/nav.scss

```diff
@@ -37,16 +37,23 @@
     &:hover {
       color: $color-primary;
     }
+    &.external-link {
+      padding-right: 1.375rem;
+      background: url('#{$static-img-path}/external-link-grey.png') right center no-repeat;
+      background: url('#{$static-img-path}/external-link-grey.svg') right center no-repeat;
+      background-size: 15px 15px;
+    }
   }
 
+
```

Comment:
Yep, our linter is set to only allow 1 blank line between declarations. If you click the `details` link on the circle ci integration below, you can see exactly what lines are problems.

I'd recommend running `npm run lint` locally after you make changes, then you can see the linting errors as they happen!